### PR TITLE
upgrade node version to 14

### LINKED ISSUE #99: Move to NodeJS 14
#### What's this issue about?

currently we use NodeJS 12, we should move to the next LTS version

https://nodejs.org/en/about/releases/

image

### PR DESCRIPTION
**Pull request by kmjayadeep**
## Description
Upgrade nodejs version to 14-alpine

## Motivation and Context
currently we use NodeJS 12, we should move to the next LTS version 14
fixes #99 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by wodka** [comment]
please also do this for the ui and api repository as here we only have the combined build

**Comment by kmjayadeep** [comment]
sure
I have changed the package.json as well accordingly. Did a sanity test and everything seems to be working fine

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "engines": {
     "yarn": "1.x",
-    "node": "12.x"
+    "node": "14.x"
```
Comment (wodka) [inline package.json:12]: please mark as `"node": "^12 || ^14"`

**Review by wodka — CHANGES_REQUESTED** [review]

**Comment by wodka** [comment]
Thank you!

**Comment by kmjayadeep** [comment]
@wodka kindly add `hacktoberfest-accepted` as the label. Otherwise it doesn't count towards the progress

**Comment by wodka** [comment]
ah, will do!
thank you

**Comment by kmjayadeep** [comment]
pull requests are made towards ui and api repos as well. Thanks for the quick response :)

**Pull request merged**